.github/workflows/release.yml

```diff
@@ -113,51 +113,11 @@ jobs:
         run: |
           gh release upload ${{ github.ref_name }} "tmp/script.ryot.zip"
 
-  docker-release:
-    runs-on: ubuntu-latest
-    needs: create-release
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-
-      - name: Log in to the Container registry
-        uses: docker/login-action@v2
-        with:
-          registry: ${{ env.REGISTRY }}
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Get docker image name and build args
-        id: required_args
-        run: |
-          image_name="${{ env.REGISTRY }}/${{ github.actor }}/${{ github.event.repository.name }}"
-          image_names="$image_name:${{ github.ref_name }},$image_name:unstable"
-          if [ "${{ needs.create-release.outputs.is-prerelease }}" = "false" ]; then
-            image_names="$image_names,$image_name:latest"
-          fi
-          # lowercase the name
-          image_names=$(echo "$image_names" | tr '[:upper:]' '[:lower:]')
-          echo "image_names=$image_names" >> $GITHUB_OUTPUT
-
-      - name: Build and push to ghcr
-        uses: docker/build-push-action@v3
-        with:
-          context: .
-          platforms: linux/amd64,linux/arm64
-          push: true
-          tags: ${{ steps.required_args.outputs.image_names }}
-
   # Mark the Github Release™ as a non-draft now that everything has succeeded!
   publish-release:
     # Only run after all the other tasks, but it's ok if upload-artifacts was skipped
     needs:
-      [create-release, upload-artifacts, docker-release, upload-kodi-plugin]
+      [create-release, upload-artifacts, upload-kodi-plugin]
```

Comment:
Should we do away with cargo-dist in this release?

Reply:
sorry, what do you mean by "do away" ?

Reply:
Oh, I got it, never heard it before. The reason why I haven't touch it is because I don't have enought information about what it is doing, espcially the `upload-kodi-plugin`. Also the deploy and the build docs should probably be kept so what do you think about these two jobs ?

Reply:
Yep `deploy` updates the public instance and `upload-kodi-plugin` creates the kodi archive. Both of the jobs are required.